src/components/inline-link/docs.mdx

```diff
@@ -15,79 +15,46 @@ Internally, this component uses two other components:
 | ------------ | -------------------------------------------------------------------------------- | --------- | ------------- | ------------------- |
 | `className`  | A string of one or more class names. Applied last to the rendered `<a>` element. | No        | undefined     | string              |
 | `href`       | The destination of the link.                                                     | Yes       | N/A           | string              |
-| `text`       | The text rendered within the `<a>` element.                                      | Yes       | N/A           | string              |
-| `textSize`   | The `size` passed to the inner `Text` component.                                 | No        | undefined     | TextProps[`size`]   |
-| `textWeight` | The `weight` passed to the inner `Text` component.                               | No        | undefined     | TextProps[`weight`] |
+| `children`   | The anchor text to render within the link.                                       | Yes       | N/A           | ReactNode           |
```

Comment:
🚧 Can this be updated for accuracy and to still mention `<a>` element? The latter was originally done to make it clear the exact element that is rendered.

```suggestion
| `children`   | The content to render within the `<a>` element.                                  | Yes       | N/A           | ReactNode           |
```